docs/azure/azure-how-to/instances/find-ubuntu-images.rst

```diff
@@ -164,7 +164,7 @@ Ubuntu Pro 24.04 LTS - Noble Numbat
 Ubuntu Pro 22.04 LTS - Jammy Jellyfish
 ++++++++++++++++++++++++++++++++++++++
 
-Quick start: `Ubuntu Pro 22.04 LTS on Azure <https://portal.azure.com/#create/canonical.0001-com-ubuntu-pro-jammypro-22_04-lts>`_
+| Quick start: `Ubuntu Pro 22.04 LTS on Azure <https://portal.azure.com/#create/canonical.ubuntu-22_04-ltsubuntu-pro>`_
```

Comment:
Did you intend to add the pipe character to the beginning of this line?

Reply:
I think I had copied this from another line, it doesn't affect rendering but is only needed when there are two or more quick starts listed. I have removed it.